src/components/QueryDateBefore/index.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryDateBefore predicate checks that the value in the path
+* is before the date value passed into the predicate.
```

Comment:
same ^^ remove ```into the predicate``` part. Also remove this for all component. Here does not exist predicate. What they call predicate now is a component and the path is the path prop passed to the component and the value is the value prop.